README.md

```diff
@@ -44,7 +44,7 @@ The migration rules use scalafix. Please see the [official installation instruct
 ```scala
 // project/plugins.sbt
 
-addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.9.0")
+addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.9.5")
```

Comment:
I'd like someone other than me to confirm this change. I can't imagine any reason it would be bad, but I don't know much about scalafix.